lib/src/utils/permissions.dart

```diff
@@ -60,3 +65,14 @@ Future<T> secureScopedAction<T>(
 
   return value;
 }
+
+Future<bool> isAndroidSdkBiggerThan33() async {
```

Comment:
Nit: maybe "isAndroidSdk33OrGreater", the version 33 would also make the method return true

Reply:
that’s a better name for sure :)

Reply:
545e620099dc5702ac423b672d88c0c912b89699